deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -266,14 +267,13 @@ async def create_new_job(
             "start_time",
             "end_time",
         ]
-        # TODO could make this if condition `details.version <= "0.25.0"` using the semver package
-        if details.version != "0.24.6":
+        if details.version.split(".") >= VersionInfo.parse("0.25.0"):
             # max_y param was added in 0.25.0
             params.append("max_y")
-        if details.version not in (
-            "0.24.6",
-            "0.25.1",
-        ):
+        if details.version.split(".") >= VersionInfo.parse("0.25.4"):
```

Comment:
👍

Reply:
The last thing you need to do is bump the pulse3d version in the worker to this version and add the scaledjob for it in the manifests